### ISSUE TITLE
DTSAPI: Duplicate reference in alignment group

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alignment-editor-new#257]
tested in Alpheios Translation Alignment editor 1.1.0 (build dev.20210303352)
when I upload a single reference, the reference is repeated twice. e.g. go to Alpheios DTS API Classical Latin Catullus Carmina Carmina and select reference 1. the alignment group will duplicate the reference, note that in origin text is not duplicated.
This is a new issue. maybe a side effect of #239 ?

![Screen Shot 2021-03-03 at 3 07 27 PM](https://user-images.githubusercontent.com/41396793/109825286-f7f5e700-7c39-11eb-91df-684175fb359b.png)

**Comment by irina060981** [comment]
>This is a new issue. maybe a side effect of #239 ?

I tested this request with outside Postman, and now our tokenizer duplicates results for Alpheios TEI XMLs.
That's why I believe that it is a side effect of that changes.
I will point it in email to @balmas at Friday

**Comment by monzug** [comment]
Thanks. I'll assign to Bridget too.

**Comment by monzug** [comment]
I don't think this is happening in Beta maṣāḥǝft DTS API but I am not able to retest as I cannot load their resources at the moment.
